File: Data Structures & Algorithms/majority-element-ii/submission-1.java
class Solution {
    public List<Integer> majorityElement(int[] nums) {
        int n = nums.length;
        int t = n / 3;
        Map<Integer, Integer> mp = new HashMap<>();
        List<Integer> res = new ArrayList<>();
        for(int x : nums) {
            mp.put(x, mp.getOrDefault(x, 0) + 1);
        }
        for(Map.Entry<Integer, Integer> it : mp.entrySet()) {
            if(it.getValue() > t) {
                res.add(it.getKey());
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        vector<int> res;
        for(int num : nums) {
           if(num != val) res.push_back(num);
        }
        nums.assign(res.begin(), res.end());
        return res.size();
    }
};